.github/workflows/plugin-release.yml

```
@@ -0,0 +1,145 @@
+      with:
+        dotnet-version: '9.0.x'
+    
+    - name: Get version from Directory.Build.props
+      id: get_version
+      run: |
+        $xml = [xml](Get-Content Directory.Build.props)
+        $version = $xml.Project.PropertyGroup.Version
+        echo "VERSION=$version" >> $env:GITHUB_OUTPUT
+        echo "Version: $version"
+      shell: pwsh
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)